code/edgemedia/src/androidTest/java/com/adobe/marketing/mobile/util/JsonTestUtils.java

```diff
@@ -0,0 +1,30 @@
+/*
+  Copyright 2023 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.Map;
+
+public class JsonTestUtils {
+
+    @SuppressWarnings("unchecked")
+    public static Map<String, Object> parseNetworkResponseAsMap(String body) {
```

Comment:
Created this Java class as a workaround to the build error "Cannot fit requested classes in a single dex file (# methods: 70537 > 65536)".  By using the Jackson Databind ObjectMapper in Java, I could remove the dependency on `jackson-module-kotlin` and therefore reduce the number of methods.